wallaby.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (wallaby) {
```

Comment:
I don't think so, unless we can get open source license for the team. but what does this force on open source users who want to run our tests?